jupyterlab_pullrequests/managers/gitlab.py

```diff
@@ -252,7 +250,7 @@ async def list_prs(self, username: str, pr_filter: str) -> List[Dict[str, str]]:
             )
 
         # Reset cache
-        self._merge_requests_cache = {}
+        self._merge_requests_cache = None
```

Comment:
Why do you reset the cache to `None` and not an empty dictionary? The idea to set as an empty dictionary is to allow to use the get method line 455.

Reply:
might be a screwup...